contracts/NFTs/VehicleId.sol

```diff
@@ -74,6 +74,14 @@ contract VehicleId is Initializable, MultiPrivilege {
         trustedForwarders[addr] = trusted;
     }
 
+    /// @notice Function to burn a token
+    /// @dev Caller must have the burner role
+    /// @dev To be called by DIMORegistry in burnVehicleSign function
```

Comment:
Is it bad that we don't allow the user themselves to call this standard ERC-721 function? I feel like it's a little bad.